src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -234,6 +234,22 @@ def add_records(
     """
     url = "/api/v1/datasets/{id}/records".format(id=id)
 
+    for record in records:
+        cleaned_responses = []
+        response_without_user_id = False
+        for response in record["responses"]:
+            if response["user_id"] is None:
+                if response_without_user_id is True:
```

Comment:
If `response_without_user_id` is always going to hold a `bool` value, then maybe we can remove the `is True` from the condition